src/batch-transaction.ts

```diff
@@ -157,7 +182,7 @@ class BatchTransaction extends Snapshot {
             client: 'SpannerClient',
             method: 'partitionQuery',
             reqOpts,
-            gaxOpts: query.gaxOptions,
+            gaxOpts: (query as ExecuteSqlRequest).gaxOptions,
```

Comment:
remove the type cast once above recommendation is implemented

Reply:
done